src/requests/workspace.jl

```diff
@@ -50,19 +50,21 @@ function process(r::JSONRPC.Request{Val{Symbol("workspace/didChangeWatchedFiles"
                 parse_all(doc, server)
             end
         elseif change.type == FileChangeTypes["Deleted"]
-            doc = getdocument(server, URI2(uri))
+            if hasdocument(server, URI2(uri))
```

Comment:
@ZacLN this diff for some reason looks more involved than it should. The only real change here is that I added this `hasdocument` check.